data/en/products.yml

```diff
@@ -26,6 +26,20 @@ products:
           link: /ensign-pricing
         - text: SDKs
           link: https://ensign.rotational.dev/sdk/
+    - product_name: Envoy
+      description: An open source peer-to-peer messaging platform built to help compliance teams handle travel rule exchanges efficiently and securely. 
```

Comment:
I used copy from the TRISA docs site about Envoy for the description and text below.